seed/seed.go

```diff
@@ -61,6 +61,10 @@ func (s *Snap) ID() string {
 	return s.SideInfo.SnapID
 }
 
+func (s *Snap) Info() snap.Info {
```

Comment:
if this returned a snap.PlaceInfo it would be less misleading

Reply:
sure, that's easy to change, also stop working on your day off! :smiley:

Reply:
it's all kind of gdocs fault :smile: